env0/resource_team_project_assignment.go

```diff
@@ -57,20 +58,25 @@ func resourceTeamProjectAssignmentRead(ctx context.Context, d *schema.ResourceDa
 		return diag.Errorf("could not get TeamProjectAssignment: %v", err)
 	}
 
-	found := false
```

Comment:
I don't see a reason why this is needed.
Specifically, why d.IsNewResource is needed.